pkg/syncer/syncer.go

```diff
@@ -327,10 +327,6 @@ func getAllGVRs(ctx context.Context, discoveryClient discovery.DiscoveryInterfac
 				// foo/status, pods/exec, namespace/finalize, etc.
 				continue
 			}
-			if !ai.Namespaced {
```

Comment:
This is not sufficient here.
We have to filter this out per GVR in the syncer as well. Provide a list somewhere with the cluster-wide GVRs allowed to be synced, and use it here.
At start we would only have `PV`s afaict.

Reply:
I'm doing the check in the resource scheduler controller:

https://github.com/kcp-dev/kcp/pull/1998/files#diff-ab772058f9afad6eb46abc8f6827d7eabb50a6c57c6afc661b4e3dca2d71013fR386-R388

Only scheduling (labeling for the syncer) only the allowed list of GVRs (PVs), keeping the syncer dumb, but I can add the check here too.

Reply:
I'd rather have the list also checked here. So that adding the label manually cannot trigger unexpected syncs of cluster-wide resources, even if it's in a short period of time.

Reply:
Maybe just factorize the list and use it in several places

Reply:
done